adapters/celestia/src/da_service.rs

```diff
@@ -126,9 +131,12 @@ impl DaService for CelestiaService {
                     .parse()
                     .unwrap(),
             );
+
             jsonrpsee::http_client::HttpClientBuilder::default()
```

Comment:
Is it possible to separate the `client` building logic (and tx sending) to a separate file? This way we will be able to reuse it more easily.